src/webui/www/private/scripts/client.js

```diff
@@ -97,13 +97,11 @@ window.addEvent('load', function() {
         placement: 'left',
         onResize: saveColumnSizes,
         width: filt_w,
-        resizeLimit: [100, 300]
+        resizeLimit: [1, 300]
```

Comment:
We have to use a minimum of `1`. Mocha treats `0` as it not being set and falls back to `50`.